library/CHART-TEMPLATE/tests/cl-region.yaml

```diff
@@ -0,0 +1,56 @@
+suite: test region helper
+templates:
+  - templates/example-cm-region.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+tests:
+  - it: Is US by default
+    asserts:
+      - equal:
+          path: data.region
+          value: US
+
+  - it: Is US with everything null
+    set:
+      global: null
+      region: null
+    asserts:
+      - equal:
+          path: data.region
+          value: US
+
+  - it: Set region (globally)
+    set:
+      global:
+        region: EU
+    asserts:
+      - equal:
+          path: data.region
+          value: EU
+
+  - it: Set region (locally)
+    set:
+      region: EU
+    asserts:
+      - equal:
+          path: data.region
+          value: EU
+
+  - it: Overrides global region with the local one
+    set:
+      global:
+        region: EU
+      region: local
+    asserts:
+      - equal:
+          path: data.region
+          value: Local
```

Comment:
Should we add another test case to check that `EU` region is inferred from the api-key when region is not set but the the api-key is available?

Reply:
Good point. I completely forgot. Done [#27eb921](https://github.com/newrelic/helm-charts/pull/1463/commits/27eb92198bd310c1c9136893e9d50a56173e4109)